docs/src/index.md

```diff
@@ -1,91 +1,74 @@
 # KernelAbstractions
 
-`KernelAbstractions.jl` is a package that allows you to write GPU-like kernels that
-target different execution backends. It is intended to be a minimal, and performant
-library that explores ways to best write heterogenous code.
+[`KernelAbstractions.jl`](https://github.com/JuliaGPU/KernelAbstractions.jl) (KA) is
+a package that allows you to write GPU-like kernels targetting different
+execution backends. KA intends to be a minimal and
+performant
+library that explores ways to write heterogeneous code. Although parts of
+the package are still experimental, it has been used successfully as part of the
+Exascale Computing Project to run Julia code on pre-Frontier and pre-Aurora
```

Comment:
Maybe add links to wikipedia articles for Aurora / Fronteir so people who are not knowledgeable in HPC know what these are?

https://en.wikipedia.org/wiki/Exascale_computing#United_States
https://en.wikipedia.org/wiki/Frontier_(supercomputer)
https://en.wikipedia.org/wiki/Aurora_(supercomputer)

As a side note: there are a bunch of other projects that use KA. Maybe it would be a good idea to have a separate section for "projects using KA"? This would allow us to link to the projects and give a blurb about them, maybe? That might be too much for KA docs, though.

I know it's been used in climate modelling, sciml, computer graphics, and I am working on MD as well.

Reply:
I added the respective official web pages. Let me know if you prefer Wikipedia.